test/error.test.js

```diff
@@ -136,30 +142,44 @@ describe('seneca-error', function () {
     test_action(si, ctxt)
   }
 
+  // REMOVE after Seneca 3.x
```

Comment:
we need a list of feature EOLs for the various versions, also covering entity etc